mlir/lib/Dialect/Rock/Pipelines/Pipelines.cpp

```diff
@@ -200,7 +200,7 @@ void rock::buildBackendPipeline(OpPassManager &pm,
   floatEmuOpts.targetTypeStr = "f32";
   gpuPm.addPass(arith::createArithEmulateUnsupportedFloats(floatEmuOpts));
   gpuPm.addPass(createLowerGpuOpsToROCDLOpsPass(
-      options.chip, options.indexBitwidth, /*useBarePtrCallConv=*/true));
+      options.chip, /*indexBitwidth=*/0, /*useBarePtrCallConv=*/true));
```

Comment:
Why is this set to zero now? might worth a comment here for future us.

Reply:
Updated to use the actual name of the constant